netmosquito.py

```diff
@@ -0,0 +1,22 @@
+import socket
+
+host='127.0.0.1' #Host IP goes here
+port=7000		 #Port Goes Here
+
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+s.connect(host,port)
+
+def recieve_data(s):
```

Comment:
This function does not receive any data < 2048. This needs a better receive function

Reply:
mispelled `receive`